GOLGI/donations.yml

```diff
@@ -0,0 +1,23 @@
+- ticker: GOLGI
+
+  recipient: 
+    name: Centre for Regenerative Medicine “Stefano Ferrari”
+    address: Via Glauco Gottardi 10 41125 Modena (Italy)
+    tel: +39.059.205811
+    contact: centromedicinarigenerativ@unimore.it
+
+  ada: 180 # <- These must be numbers. Can be decimals though. See below
+  usd: 212.5 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1625468400 
+  # ^ This is a unix timestamp of the moment you donated.
+  #   You can go to https://www.unixtimestamp.com/ to calculate your donation.
+
+  files:
+  # ^ a list of files relating to the donation. Usualy photos or screenshots 
+  #   of the receipts.
+  #   Make sure to add only relative paths that begin from the root of the 
+  #   repo (they should include your Ticker name).
+    - GOLGI/20210705_first donation Golgi Stake Pool.jpg
```

Comment:
Sorry to bother you again. It seems the GitHub checks in place are not working with spaces in files names.
Could you please rename this file and the reference here. 
Unfortunately, I cannot approve as long as the automated checks are not passing.
Sorry for the inconvenience.

Reply:
Hello,
no problem I uploaded new file with no spaces in the name and I have modified the .yml according to new filename..

I hope I understand correctly your advice.

Thanks.